src/pages/Favourite/index.jsx

```diff
@@ -19,12 +19,10 @@ const FavouritePage = () => {
 
   return (
     <div>
-      <div className="image-section">
-        <Favourites
-          photos={favouritePhotos}
-          addToFavourites={handleToggleFavourite}
-        />
-      </div>
+      <GalleryList
+        photos={favouritePhotos}
+        handleToggleFavourite={handleToggleFavourite}
```

Comment:
We can handle `handleToggleFavourite` inside the GalleryListComponent right?

Reply:
it is called in photos.

Reply:
is it ok ?

Reply:
Yes you can directly use it in the GalleryListComponent itself.

Reply:
ok